JaxrsExceptionTestHelper#assertContainsError should report actual errors

`JaxrsExceptionTestHelper#assertContainsError(response, statusCode, substring)` reports an error when a `Response` does not have the given HTTP status code and does not contain an `ErrorMessage` whose `message` contains the given `substring`. For example:

```
java.lang.AssertionError: Response does not contain an ErrorMessage having status 422 and message containing: must occur before End Time
```

However, it does _not_ provide any information about what it actually contains, which is needed to help determine what went wrong. Without this information the only way I have found is to use the debugger to inspect the actual ErrorMessage(s).

This task is to enhance the method with information about the actual ErrorMessage(s) that were found in the Response, or none if there weren't any. The information should include all the information from the ErrorMessage(s) - status code, field name, item ID, and message.

--- a/src/main/java/org/kiwiproject/test/jaxrs/exception/JaxrsExceptionTestHelper.java
+++ b/src/main/java/org/kiwiproject/test/jaxrs/exception/JaxrsExceptionTestHelper.java
@@ -119,14 +119,14 @@ public class JaxrsExceptionTestHelper {
     public static ErrorMessage assertContainsError(Response response, int statusCode, String substring) {
         var jaxrsException = toJaxrsException(response);
 
-        return jaxrsException.getErrors()
-                .stream()
+        var errors = jaxrsException.getErrors();
+        return errors.stream()
                 .filter(error -> matchesArguments(error, statusCode, substring))
                 .findFirst()
                 .orElseThrow(() -> {
                     var assertErrorMessage =
-                            f("Response does not contain an ErrorMessage having status {} and message containing: {}",
-                                    statusCode, substring);
+                            f("Response does not contain an ErrorMessage having status {} and message containing: '{}'. Actual errors: {}",
+                                    statusCode, substring, errors);
                     return new AssertionError(assertErrorMessage);
                 });
     }

--- a/src/test/java/org/kiwiproject/test/jaxrs/exception/JaxrsExceptionTestHelperTest.java
+++ b/src/test/java/org/kiwiproject/test/jaxrs/exception/JaxrsExceptionTestHelperTest.java
@@ -267,8 +267,8 @@ class JaxrsExceptionTestHelperTest {
             assertThatThrownBy(() ->
                     JaxrsExceptionTestHelper.assertContainsError(response, statusCode, substring))
                     .isInstanceOf(AssertionError.class)
-                    .hasMessage("Response does not contain an ErrorMessage having status %d and message containing: %s",
-                            statusCode, substring);
+                    .hasMessage("Response does not contain an ErrorMessage having status %d and message containing: '%s'. Actual errors: %s",
+                            statusCode, substring, jaxrsException.getErrors());
         }
     }
 }

